radio/src/targets/taranis/board.cpp

```diff
@@ -442,3 +442,30 @@ const etx_serial_port_t* auxSerialGetPort(int port_nr)
   if (port_nr >= MAX_AUX_SERIAL) return nullptr;
   return serialPorts[port_nr];
 }
+
+void setMutePin(GPIO_TypeDef* GPIOx,uint16_t Pinx)
+{
```

Comment:
please move these functions to the driver itself and declare them `static` (private to the compile unit).

Reply:
Also use only the state as parameter (for instance: `bool enabled`).

Reply:
I think this was resolved cleaner by removing the funcs and just doing the inversion based on the define